includes/admin/class-wcmp-export.php

```diff
@@ -154,7 +160,6 @@ public function admin_notices()
      */
     public function export()
     {
-//        die("request: {$_REQUEST["request"]}");
```

Comment:
//        die("request: {$_REQUEST["request"]}");

Moet weg

Reply:
Uhm dit is juist weggehaald :P